test/test_helper.rb

```diff
@@ -1,4 +1,6 @@
 $:.unshift File.expand_path('../lib', File.dirname(__FILE__))
 require 'ifconfig'
-require 'test/unit'
+require 'minitest/autorun'
 require 'pp'
+
+Test = Minitest
```

Comment:
What's the purpose of the changes to test/test_helper.rb?  They seem unrelated to the rest of the pull request.